src/Organisms/ContestDetails/ContestDetailsContent/ContestDetailsApplicationInfo/ContestDetailsApplicationInfo.js

```diff
@@ -0,0 +1,44 @@
+import ContestDetailsLine from '../../../../Atoms/ContestDetailsLine/ContestDetailsLine';
+import PropTypes from 'prop-types';
+import { getDataFormatDdMmhYyy } from '../../../../Tools/TimeFunctions';
+import { applicationInfoTextTemplate } from '../../../../Consts/ContestDetailsCONSTS';
+
+const ContestDetailsApplicationInfo = ({ applicationInfo }) => {
+  const { applicationOpenDate, applicationClosedDate, applicationFeeInPLN } =
+    applicationInfo;
+
+  const applicationInfoText = applicationInfoTextTemplate;
+
+  applicationOpenDate &&
+    (applicationInfoText[0][1] = `${getDataFormatDdMmhYyy(
+      applicationOpenDate,
```

Comment:
Perhaps some documentation could be useful to explain the usage and creation of this data, using [x][y] is not self explanatory.